tests/integration/test_sqlalchemy_integration.py

```diff
@@ -20,8 +23,16 @@
 @pytest.fixture
 def trino_connection(run_trino, request):
     _, host, port = run_trino
-    engine = sqla.create_engine(f"trino://test@{host}:{port}/{request.param}",
-                                connect_args={"source": "test", "max_attempts": 1})
+
+    engine = sqla.create_engine(
+        f"trino://test@{host}:{port}/{request.param}",
+        connect_args={
+            "max_attempts": 1,
+            "schema": "test",
```

Comment:
I needed to explicitly set the schema as part of the engine otherwise Trino barfed when trying to create the views with the `Schema must be specified when session schema is not set` error.

Reply:
On first sight it can be done by doing something like:
```
create_view(
            ...,
            schema="test")
```
See https://github.com/kvesteri/sqlalchemy-utils/blob/1f6e2d6c029f5c578be0e48c7baa98e6984b7bf8/sqlalchemy_utils/view.py#L63

I also think you could include the schema in the pytest param `@pytest.mark.parametrize('trino_connection', ['memory/test'], indirect=True)` but personally I would prefer to have it in the `create_view` call.

Reply:
@mdesmet the [create_view](https://github.com/kvesteri/sqlalchemy-utils/blob/1f6e2d6c029f5c578be0e48c7baa98e6984b7bf8/sqlalchemy_utils/view.py#L122-L127) method does not support setting an explicit schema. Regrettably also defining, 

```
metadata = sqla.MetaData(schema=schema)
``` 

did not work and thus the only solution I was able to find—I spent hours trying to get this to work—was to explicitly set the schema at the time of the engine creation.